### PR TITLE
Staking concept doc

### PR DESCRIPTION
**Pull request by rajkaramchedu**
**Update: Do not review yet. Based on initial comments I am rewriting this. Putting this PR in the Draft mode.**

This PR is for the staking concept doc. I wrote it to introduce the notion of staking at high level. Please review, thanks! 

I will create a separate PR for the staking instructions, requirements and expectations for the Validator in the AIT-2.

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/1621)

### CODE REVIEW

File: developer-docs-site/docs/nodes/staking/staking-on-aptos.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+This document presents a conceptual introduction to staking. 
+
+Staking is at the heart of the distributed consensus in the authority-less world of public blockchains. In particular, staking makes possible a particular type of distributed consensus mechanism called **proof of stake** consensus. 
+
+## Transaction integrity
+
+When you submit a transaction to a blockchain, the desired outcome is **all** of the following:
+
+1. All the events defined in the transaction occur with integrity.
+2. These events are permanently recorded in a secure and immutable way in the blockchain, and,
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/staking/staking-on-aptos.md:17]: the and is weird at the end of this, I'd drop it

File: developer-docs-site/docs/nodes/staking/staking-on-aptos.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+Some nodes, called Validators, are given temporary authority to decide if the next set of transactions in a block are correct. Only after a Validator validates these transactions will the block be included in the blockchain. 
+
+The authority is **temporary** because the same Validator cannot continue to validate transaction after transaction in block after block. Allowing the same Validator to do so will start to look like the Validator is a central authority, effectively monopolizing and controlling the distributed consensus mechanism.
+
+:::tip 
+
+Distributed consensus mechanisms are usually deployed to work only within the scope of a blockchain.
+:::
+
+### Validators enter into a draw
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/staking/staking-on-aptos.md:41]: What's a draw, dualing?

I think this is unclear from an English perspective.

File: developer-docs-site/docs/nodes/staking/staking-on-aptos.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+This document presents a conceptual introduction to staking. 
+
+Staking is at the heart of the distributed consensus in the authority-less world of public blockchains. In particular, staking makes possible a particular type of distributed consensus mechanism called **proof of stake** consensus. 
+
+## Transaction integrity
+
+When you submit a transaction to a blockchain, the desired outcome is **all** of the following:
+
+1. All the events defined in the transaction occur with integrity.
```
Comment (davidiw) [inline developer-docs-site/docs/nodes/staking/staking-on-aptos.md:16]: I don't know what this means.

**Review by davidiw — COMMENTED** [review]
I'd like to revisit what the goal of this doc is. I think it might be better to start with a very simple overview of what staking means. Then how staking drives consensus. Then what are the knobs for staking (joining, leaving, lock-in time).

**Comment by rajkaramchedu** [comment]
Do not review yet. Based on initial comments I am rewriting this in the Draft mode.

**Comment by davidiw** [comment]
poking you by doing what I do best...

**Pull request closed without merging**